docs/devguide/building/cmake.rst

```diff
@@ -938,3 +938,13 @@ Features
   :no-contents-entry:
 
   Default is ``OFF``
+
+.. cmake:var:: OPENDDS_MONOTONIC_USES_BOOTTIME
```

Comment:
I realized there's a bit of a inconstancy to this that might or might not matter.

If the user builds using the configure script using `--boottime` and then looks at this variable, it will be `FALSE`. We'd have to set it in [`configure`](https://github.com/OpenDDS/OpenDDS/blob/d139fe743445fa144a1eae230d5a2da704a8b692/configure#L2847-L2871). We could add it there, but maybe it'd make more sense to move this from here to the [build-exclusive variables](https://github.com/OpenDDS/OpenDDS/blob/d139fe743445fa144a1eae230d5a2da704a8b692/docs/devguide/building/index.rst?plain=1#L586) to better reflect how it's used.